### PR TITLE
Prep for v1.13.0

### PR DESCRIPTION
**Pull request by blegat**
We need to wait for

- [x] https://github.com/jump-dev/JuMP.jl/pull/3434

## Pre-release

 - [ ] Check that the pinned packages in `docs/Project.toml` are updated. We pin
       the versions so that changes in the solvers (changes in printing, small
       numeric changes) do not break the printing of the JuMP docs in arbitrary
       commits.
 - [ ] Check that the `rev` fields in `docs/packages.toml` are updated. We pin
       the versions of solvers and extensions to ensure that changes to their
       READMEs do not break the JuMP docs in arbitrary commits, and to ensure
       that the versions are compatible with the latest JuMP and
       MathOptInterface releases.
 - [x] Update `docs/src/changelog.md`
 - [x] https://github.com/jump-dev/JuMP.jl/actions/runs/5642902565
 - [x] Change the version number in `Project.toml`
 - [x] Update the links in README.md
 - [x] The commit messages in this PR do not contain `[ci skip]`

## The release

 - [ ] After merging this pull request, comment `[at]JuliaRegistrator register` in
       the GitHub commit. This should automatically publish a new version to the
       Julia registry, as well as create a tag, and rebuild the documentation
       for this tag.

       These steps can take quite a bit of time (1 hour or more), so don't be
       surprised if the new documentation takes a while to appear. In addition,
       the links in the README will be broken until JuliaHub fetches the new
       version on their servers.

## Post-release

 - [ ] Once the tag is created, update the relevant `release-` branch. The latest
       release branch at the time of writing is `release-1.0` (we haven't
       back-ported any patches that needed to create a `release-1.Y` branch). To
       to update the release branch with the v1.10.0 tag, do:
       ```
       git checkout release-1.0
       git pull
       git merge v1.10.0
       git push
       ```

### CODE REVIEW

File: docs/src/changelog.md [hunk 1]
```diff
@@ -7,6 +7,30 @@ CurrentModule = JuMP
...
+
+### Added
+
+ - Added support for generic number type (#3377) (#3385)
+ - Added fallback for [`MOI.AbstractSymmetricMatrixSet`](@ref) (#3424)
+
+### Fixed
+
+ - Fixed [`set_start_values`](@ref) with
+   [`MOI.Bridge.Objective.SlackBridge`](@ref) (#3422)
```
Comment (odow) [inline docs/src/changelog.md:20]:
```suggestion
   [`MOI.Bridges.Objective.SlackBridge`](@ref) (#3422)
```

File: docs/src/changelog.md [hunk 2]
```diff
@@ -7,6 +7,30 @@ CurrentModule = JuMP
...
+   [`MOI.Bridges.Objective.SlackBridge`](@ref) (#3422)
+ - Fixed flakey doctest in `variables.md` (#3425)
+ - Fixed names on `CITATION.bib` (#3423)
+
+### Other
+
+ - Added Loraine.jl to the installation table (#3426)
+ - Removed Penopt.jl from packages.toml (#3428)
+ - Improved problem statement in cannery example of tutorial (#3430)
+ - Minor cleanups in [`DenseAxisArray`](@ref) implementation (#3429)
```
Comment (odow) [inline docs/src/changelog.md:29]:
```suggestion
 - Minor cleanups in [`Containers.DenseAxisArray`](@ref) implementation (#3429)
```

File: docs/src/changelog.md [hunk 3]
```diff
@@ -7,6 +7,30 @@ CurrentModule = JuMP
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 1.13.0 (July 24, 2023)
+
+### Added
+
+ - Added support for generic number type (#3377) (#3385)
+ - Added fallback for [`MOI.AbstractSymmetricMatrixSet`](@ref) (#3424)
```
Comment (odow) [inline docs/src/changelog.md:15]:
```suggestion
 - Added fallback for [`MOI.AbstractSymmetricMatrixSetTriangle`](@ref) and
   [`MOI.AbstractSymmetricMatrixSetSquare`](@ref) (#3424)
```

File: docs/src/changelog.md [hunk 4]
```diff
@@ -7,6 +7,31 @@ CurrentModule = JuMP
...
+ - Fixed names on `CITATION.bib` (#3423)
+
+### Other
+
+ - Added Loraine.jl to the installation table (#3426)
+ - Removed Penopt.jl from packages.toml (#3428)
+ - Improved problem statement in cannery example of tutorial (#3430)
+ - Minor cleanups in [`Containers.DenseAxisArray`](@ref) implementation (#3429)
+ - Changed `nested_problems.jl`: outer/inner to upper/lower (#3433)
+ - Removed second SDP relaxation due to unreliable numerics in OPF tutorial
```
Comment (odow) [inline docs/src/changelog.md:32]:
```suggestion
 - Removed second SDP relaxation in OPF tutorial
```

File: docs/src/changelog.md [hunk 5]
```diff
@@ -7,6 +7,31 @@ CurrentModule = JuMP
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 1.13.0 (July 24, 2023)
```
Comment (odow) [inline docs/src/changelog.md:10]:
```suggestion
## Version 1.13.0 (July 25, 2023)
```

File: docs/src/changelog.md [hunk 6]
```diff
@@ -7,6 +7,31 @@ CurrentModule = JuMP
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 1.13.0 (July 25, 2023)
+
+### Added
+
+ - Added support for generic number type (#3377) (#3385)
```
Comment (mlubin) [inline docs/src/changelog.md:14]:
```suggestion
 - Added support for generic number types (#3377) (#3385)
```

**Review by mlubin — APPROVED** [review]

File: docs/src/changelog.md [hunk 7]
```diff
@@ -7,6 +7,31 @@ CurrentModule = JuMP
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 1.13.0 (July 25, 2023)
```
Comment (odow) [inline docs/src/changelog.md:10]:
```suggestion
## Version 1.13.0 (July 26, 2023)
```

File: docs/src/changelog.md [hunk 8]
```diff
@@ -7,6 +7,31 @@ CurrentModule = JuMP
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Version 1.13.0 (July 26, 2023)
```
Comment (blegat) [inline docs/src/changelog.md:10]:
```suggestion
## Version 1.13.0 (July 27, 2023)
```

**Comment by mlubin** [comment]
Are we going to merge today or not? I need to know the latest version for my talk :)

**Pull request merged**